ql/src/java/org/apache/hadoop/hive/llap/LlapHiveUtils.java

```diff
@@ -48,18 +50,27 @@ private LlapHiveUtils() {
     // Not to be used;
   }
 
-  public static CacheTag getDbAndTableNameForMetrics(Path path, boolean includeParts,
-        Map<Path, PartitionDesc> parts) {
-
-    assert(parts != null);
+  /**
+   * Takes path and a map of partition descs, returns the entry associated with the given path.
```

Comment:
This comment could be optimized. The logic isn't really explained well and that parameter description doesn't help much either.